website/client/js/controllers/rootCtrl.js

```diff
@@ -24,8 +24,10 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
     $rootScope.$on('$stateChangeSuccess',
       function(event, toState, toParams, fromState, fromParams){
         $rootScope.pageTitle = $state.current.title;
-
-        $window.scrollTo(0, 0);
+        
+        if ($rootScope.pageTitle != 'Challenges') {
```

Comment:
The page title is actually a translation string. So this won't work for non English users.